test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -52,30 +76,436 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, func() {
+				ovnPod, err := getOvnPod(workerRTNode)
+				Expect(err).ToNot(HaveOccurred(), "Unable to get ovnPod")
+
+				ovnContainersids, err := getOvnPodContainers(&ovnPod)
+				Expect(err).ToNot(HaveOccurred())
+
+				// Generally there are many containers inside a kubenode pods
+				// we don't need to check cpus used by all the containers
+				// we take first container
+				cpus := getCpusUsedByOvnContainer(workerRTNode, ovnContainersids[0])
+				testlog.Infof("Cpus used by ovn Containers are %s", cpus)
+				pidList, err := getOVSServicesPid(workerRTNode)
+				Expect(err).ToNot(HaveOccurred())
+
+				for _, pid := range pidList {
+					cmd := []string{"taskset", "-pc", pid}
+					cpumask, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+					Expect(err).ToNot(HaveOccurred(), "failed to fetch cpus of %s", pid)
+					mask := strings.SplitAfter(cpumask, " ")
+					maskSet, err := cpuset.Parse(mask[len(mask)-1])
+					testlog.Infof("cpus used by ovs services are: %s", maskSet.String())
+					Expect(cpus).To(Equal(maskSet.String()), "affinity of ovn kube node pods(%s) do not match with ovservices(%s)", cpus, maskSet.String())
+					if err != nil {
+						testlog.Error(err.Error())
+					}
+				}
+			})
+
+			It("[test_id:64101] Creating gu pods modifies affinity of ovs", func() {
+				var testpod *corev1.Pod
+				var err error
+				testpod = pods.GetTestPod()
+				testpod.Namespace = testutils.NamespaceTesting
+				testpod.Spec.Containers[0].Resources = corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("2"),
+						corev1.ResourceMemory: resource.MustParse("200Mi"),
+					},
+				}
+				testpod.Spec.NodeSelector = map[string]string{testutils.LabelHostname: workerRTNode.Name}
+				err = testclient.Client.Create(context.TODO(), testpod)
+				Expect(err).ToNot(HaveOccurred())
+				testpod, err = pods.WaitForCondition(client.ObjectKeyFromObject(testpod), corev1.PodConditionType(corev1.PodReady), corev1.ConditionTrue, 5*time.Minute)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(testpod.Status.QOSClass).To(Equal(corev1.PodQOSGuaranteed))
+
+				cmd := []string{"taskset", "-pc", "1"}
+				testpodCpus, err := pods.ExecCommandOnPod(testclient.K8sClient, testpod, cmd)
+				testlog.Infof("%v pod is using %v cpus", testpod.Name, string(testpodCpus))
+
+				By("Get ovnpods running on the worker cnf node")
+				ovnPod, err := getOvnPod(workerRTNode)
+				Expect(err).ToNot(HaveOccurred(), "Unable to get ovnPod")
+
+				By("Get cpu used by ovn pod containers")
+				ovnContainers, err := getOvnPodContainers(&ovnPod)
+				Expect(err).ToNot(HaveOccurred())
+				cpus := getCpusUsedByOvnContainer(workerRTNode, ovnContainers[0])
+
+				pidList, err := getOVSServicesPid(workerRTNode)
+				Expect(err).ToNot(HaveOccurred())
 
-			ovsCPUSet, err := cpuset.Parse(ovsCpuList)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(ovsCPUSet).To(Equal(onlineCPUSet))
+				for _, pid := range pidList {
```

Comment:
And another here, the ovnkube-node only checks its affinity once per a second (used to be 10 seconds).